src/app.ts

```diff
@@ -141,7 +141,10 @@ export default class CeramicAnchorApp {
     const ipfsService: IpfsServiceImpl = container.resolve<IpfsServiceImpl>('ipfsService');
     await ipfsService.init();
 
-    await this._executeAnchor();
+    this.startWithConnectionHandling(async () => {
```

Comment:
if we don't wait for the anchor to finish, won't we terminate the process before the anchor has run?

Reply:
No bc the event loop will not be finished until the anchor is complete.

Code sample similar to what we do:
```js
async function stall() {
  await _start()
  console.log('stall')
}

async function _start() {
  console.log('_start')
  _nonBlocking()
}

async function _nonBlocking() {
  console.log('_nonBlocking')
  setTimeout(() => console.log('timed out'), 1000)
}

stall()
```

I.e. `await` is blocking relative to other code in the program, but the absence of it does not terminate the process